src/Panes/AppearancePane.vala

```diff
@@ -31,47 +31,88 @@ public class PantheonTweaks.Panes.AppearancePane : Categories.Pane {
     private int controls_index = 0;
 
     public AppearancePane () {
-        base (_("Appearance"), "applications-graphics");
+        base (
+            _("Appearance"), "applications-graphics",
+            _("Change the theme and button layout of windows. Changing theme may cause visibility issue.")
+        );
     }
 
     construct {
         var theme_label = new Granite.HeaderLabel (_("Theme Settings"));
-        var theme_box = new Widgets.SettingsBox ();
 
+        var gtk_label = new SummaryLabel (_("GTK+:"));
         var gtk_store = Util.get_themes_store ("themes", "gtk-3.0", InterfaceSettings.get_default ().gtk_theme, out gtk_index);
-        gtk_combobox = theme_box.add_combo_box (_("GTK+"));
+        gtk_combobox = new ComboBox ();
         gtk_combobox.set_model (gtk_store);
 
+        /// TRANSLATORS: The two "%s" represent the paths to put custom icons
+        var gtk_info = new DimLabel (_("To show custom themes here, put them in %s.").printf (
+            "~/.local/share/themes/<theme-name>/gtk-3.0"
+        ));
```

Comment:
Sorry to nitpick, but `<theme-name>` would still show up in English and be non translatable. The rest of the path is constant regardless of the language, right? If so i think that we could have something like this to prevent the translators from having to translate the path multiple times :)
```suggestion
        var gtk_info = new DimLabel (_("To show custom themes here, put them in %s.").printf (
            "~/.local/share/themes/<%s>/gtk-3.0".printf(_("theme-name"))
        ));
```

Reply:
Ah, that's what you mean in the previous review. Thank you for detailed explanation!